Sources/Apodini/Components/Component.swift

```diff
@@ -16,7 +15,7 @@ public protocol Component {
     /// The type of `Component` this `Component` is made out of if the compoent is a composition of multiple subcomponents.
     associatedtype Content: Component = Never
     /// The type that is returned from the `handle` method when the component handles a request. The returntyp of the `handle` method is encoded into the response send out to the client.
-    associatedtype Response: ResponseEncodable = Never
+    associatedtype Response: Encodable = Never
```

Comment:
We might want to consider a different protocol that will give us the information we need for HATEOAS + Documentation for OpenAPI

Reply:
See #41